### PR TITLE
Tuto vae

### PR DESCRIPTION
**Pull request by OlivierBernard**
Let's start the show ;)

### CODE REVIEW

File: collections/_tutorials/2022-09-12-tutorial-vae.md [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+
+![](/collections/images/vae/vae_latent_space_gaussian.jpg)
+
+>>This aspect of VAE can be seen as manifold learning
+
+&nbsp;
+
+1. Let's say we have a distribution $$p(z/x)$$ that we don't know
+
+2. We use a new (simpler) distribution $$q(z)$$ to estimate $$p(z/x)$$
```
[the remark this answers is not in the document]
Comment (jodoin) [reply collections/_tutorials/2022-09-12-tutorial-vae.md:142]: aussi, personnellement, j'écrirais ici q(z/x) et non q(z).

File: collections/_tutorials/2022-09-12-tutorial-vae.md [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+
+$$D_{KL}\left(p \parallel q \right) = H_{pq} - H_{p}$$
+
+$$D_{KL}\left(p \parallel q \right) = -\int{p(x)\cdot log\left(q(x)\right)}\,dx + \int{p(x)\cdot log\left(p(x)\right)}\,dx$$
+
+$$D_{KL}\left(p \parallel q \right) = -\int{p(x)\cdot log\left(\frac{q(x)}{p(x)}\right)}\,dx$$
+
+$$D_{KL}\left(p \parallel q \right) = \int{p(x)\cdot log\left(\frac{p(x)}{q(x)}\right)}\,dx$$
+
+KL divergence allows to measure a distance between two distributions with the following properties:
```
Comment (jodoin) [inline collections/_tutorials/2022-09-12-tutorial-vae.md:106]: positive 

because Hpq>Hp

**Review by jodoin — CHANGES_REQUESTED** [review]

**Review by OlivierBernard — COMMENTED** [review]
All the comments have been taking into account

**Comment by OlivierBernard** [comment]
@Nathan: I am not still 100% happy with my post, I will integrate others concepts to strengthen it ! Let's wait for my next commit before doing your review

**Review by jodoin — APPROVED** [review]

**Review by nathanpainchaud — COMMENTED** [review]
Great tutorial! Just a few small comments related to typos/formatting ;)

Also, I haven't looked through all the figures you've produced for the tutorial, but some (e.g. _decoder_illustration_ and _decoder_illustration_2_ ) are not used in the tutorial anymore. Maybe they were in previous WIP versions? So I would suggest you delete the unused figures to avoid taking up unnecessary space on the repo.

**Pull request merged**